akka-actor-typed-tests/src/test/scala/akka/actor/typed/SupervisionSpec.scala

```diff
@@ -261,7 +261,7 @@ class SupervisionSpec extends ScalaTestWithActorTestKit("""
   // FIXME #24348: eventfilter support in typed testkit
   import akka.actor.typed.scaladsl.adapter._
 
-  implicit val untypedSystem = system.toUntyped
+  implicit val classicSystem = system.toClassic
```

Comment:
These will likely conflict with the logging changes since it shouldn't be needed anymore with the typed log event filters in place.

Reply:
yes, I have removed all those, but it will conflict no matter how we do it. I'll deal with that in my PR

Reply:
Anything I can/should do differently here for that @johanandren @patriknw ?

Reply:
no, it's good here. It will become conflicts in my PR, but that's just fine